Swift/BRCrypto/BRCryptoTransfer.swift

```diff
@@ -236,12 +235,18 @@ public class TransferFeeBasis {
     deinit {
         cryptoFeeBasisGive (core)
     }
+
+    public static func == (lhs: TransferFeeBasis, rhs: TransferFeeBasis) -> Bool {
```

Comment:
What about a C level `cryptoFeeBasisIsIdentical()`?

Reply:
Added this (and associated ETH version).